.prettierrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
```

Comment:
I would suggest removing this config, and move it to `.eslint.json`. Then running prettier through ESLint as a plugin. This way we have a single tool for linting/formatting (ESLint formats as well with `--fix`, and if the two have conflicting rules or runs in parallel, "fun" stuff can happen).

It's also easier to setup editors. vscode example:
1) install [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) 
2) Add the following to `settings.json`:
```json
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  }
```

Reply:
I though this was the recommended way of configuring Prettier, even if invoked via ESLint: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

Reply:
It is, but afaik it's with the soul focus of letting tools pick up that prettier is "active" in the project, for IDE extensions/tooling. This is also the reason for me to suggest moving it - as running eslint & prettier simultaneously will produce unwanted results (in case you have both prettier & eslint extensions installed).
Might not be an issue.